crates/goose/examples/image_tool.rs

```diff
@@ -1,6 +1,6 @@
 use anyhow::Result;
 use base64::{engine::general_purpose::STANDARD as BASE64, Engine as _};
-use dotenv::dotenv;
+use dotenvy::dotenv;
```

Comment:
nice - yeah just read the advisory, and dotenv hasn't had an update for 5 years (!). dotenvy is 5 months (which is ok I guess)